vars/deployClusterPipeline.groovy

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/groovy
+
+def call(String jobName, String stableNamespace, String stableImageTag, Map<String,String> tagOverrides) {
+  pipeline {
+    agent any
+
+    parameters {
+      string(name: "BUILD_ID")
+      string(name: "JOB_SPEC")
+      string(name: "REPO_OWNER")
+      string(name: "REPO_NAME")
+      string(name: "PULL_REFS")
+      string(name: "PULL_BASE_REF")
+      string(name: "PULL_BASE_SHA")
+      string(name: "PULL_NUMBER")
+      string(name: "PULL_PULL_SHA")
+    }
+
+    stages {
+      stage ("Configure Variables") {
+        steps {
+          script {
+            this._buildName = buildName(this)
+            this._jobId = "${jobName}-${this._buildName}-${env.BUILD_NUMBER}"
+          }
+        }
+      }
+      stage ("Ensure logging components exist") { steps {
+          ensureLoggingComponents(this)
+        }
+      }
+      stage("Extract Config Vars") {
+        steps {
+          script {
+            openshift.withCluster() {
+              def cm = openshift.selector("cm/${this._buildName}").object()
+              this._imageBase = cm.data["image-base"]
```

Comment:
This is the namespace where we're pulling component images from?

Reply:
it's the value we pass to the origin installer for the base origin image ref (registry.ip/namespace/origin:ci)